docs/topics/api/addons.rst

```diff
@@ -53,7 +53,7 @@ This endpoint allows you to search through public add-ons.
            hotness  Hotness (average number of users progression), descending.
             random  Random ordering. Only available when no search query is
                     passed and when filtering to only return promoted add-ons.
-            rating  Bayesian rating, descending.
+           ratings  Bayesian rating, descending.
```

Comment:
I think it's confusing enough that we can document both (indicating that old one is deprecated/only there for backwards-compatibility), in case people stumble upon old code using `rating`.

Reply:
Agreed, fixed:

```diff
-            rating  Bayesian rating, descending.
+           ratings  Bayesian rating, descending (preferred).
+            rating  Bayesian rating, descending (deprecated, backwards-compatibility only).
```

Reply:
The deprecated warning is fine, but I don't think we need the `(preferred)` bit on the other one.

Reply:
Done